packages/apps/tools/src/components/Global/ModuleExplorer/index.tsx

```diff
@@ -0,0 +1,41 @@
+import type { ChainwebChainId } from '@kadena/chainweb-node-client';
+
+import type { IEditorProps } from './editor';
+import Editor from './editor';
+import SidePanel from './SidePanel';
+import { containerStyle } from './styles.css';
+import { getModulesMap } from './utils';
+
+import React from 'react';
+
+export interface IModule {
+  chainId: ChainwebChainId;
+  moduleName: string;
+}
+
+export interface IModuleExplorerProps {
+  modules: IModule[];
+  openedModules: IEditorProps['openedModules'];
+  onModuleClick: (module: IModule) => void;
+}
+
+const ModuleExplorer = ({
+  modules,
+  openedModules,
+  onModuleClick,
+}: IModuleExplorerProps) => {
+  const results = getModulesMap(modules);
+  return (
+    <div className={containerStyle}>
```

Comment:
Is it possible here to use the Grid from the react-ui lib (Grid.Root & Grid.Item) instead of a regular div?

Reply:
Unfortunately, I can't, since the Grid only supports columns of equal sizes, which is not the case for me 😁